egs/gigaspeech/ASR/zipformer/run_train_cluster.sh

```
@@ -0,0 +1,44 @@
+#
+# Launch zipformer training with torchrun (single- or multi-node).
+# train.py reads RANK / LOCAL_RANK / WORLD_SIZE from the env, so
+# `--world-size` is ignored here.
+#
+# Single 8-GPU node:  bash zipformer/run_train_cluster.sh
+# Multi-node: the scheduler is expected to export WORLD_SIZE (#nodes),
+# RANK (node rank), MASTER_ADDR and MASTER_PORT.
+
+set -euo pipefail
+
+export PYTHONPATH=`pwd`/../../../
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)